test/extensions/filters/http/ext_authz/ext_authz_fuzz.proto

```diff
@@ -1,29 +1,44 @@
 syntax = "proto3";
 package envoy.extensions.filters.http.ext_authz;
 
+import "envoy/config/core/v3/base.proto";
 import "envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto";
+import "envoy/service/auth/v3/external_auth.proto";
 import "test/fuzz/common.proto";
-import "envoy/config/core/v3/base.proto";
 import "validate/validate.proto";
 
 // We only fuzz a single request per iteration.
-message ExtAuthzTestCase {
+message ExtAuthzTestCaseBase {
+  envoy.extensions.filters.http.ext_authz.v3.ExtAuthz config = 1
+      [(validate.rules).message = {required: true}];
+  // HTTP request data.
+  test.fuzz.HttpData request_data = 2 [(validate.rules).message = {required: true}];
+  // Filter metadata.
+  envoy.config.core.v3.Metadata filter_metadata = 4;
+}
+
+message ExtAuthzTestCaseGrpc {
+  ExtAuthzTestCaseBase base = 1 [(validate.rules).message = {required: true}];
+
+  oneof response_or_failure_reason {
+    // Full auth check result. Note it is not validated to simulate an untrusted authz server (i.e.
+    // it can contain garbage mutations).
+    envoy.service.auth.v3.CheckResponse response = 2 [(validate.rules).message.skip = true];
```

Comment:
Good to add the CheckResponse filed in the test case. It enables fuzzing the response.